src/map-caption.js

```diff
@@ -0,0 +1,45 @@
+import './leaflet.js'; 
```

Comment:
Lines 1 & 2 aren't necessary, because you don't use any symbols from those imports.